src/System.Net.HttpListener/src/System/Net/HttpResponseStream.cs

```diff
@@ -40,6 +40,19 @@ public override IAsyncResult BeginRead(byte[] buffer, int offset, int size, Asyn
         }
 
         public override int EndRead(IAsyncResult asyncResult) => throw new InvalidOperationException(SR.net_writeonlystream);
+        
+        public new Task<int> ReadAsync(Byte[] buffer, int offset, int count)
```

Comment:
I explained why can't use `override` here: https://github.com/dotnet/corefx/issues/22066#issuecomment-315464221

The compiler will complain.